.github/dependabot.yml

```
@@ -42,6 +42,9 @@ updates:
       # @see https://github.com/dequelabs/axe-core/issues/4428
       - dependency-name: 'colorjs.io'
         versions: ['>0.4.3']
+      # Still need to support node 18
+      - dependency-name: 'glob'
+        versions: ['>11.0.0']
```

11.0.0 is the first version that requires node 20+ and not the last version that supports node 18, right?

```suggestion
        versions: ['>=11.0.0']
```